addons/mail/static/tests/chatter_tests.js

```diff
@@ -113,7 +113,9 @@ QUnit.test('list activity widget with no activity', async function (assert) {
         data: this.data,
         arch: '<list><field name="activity_ids" widget="list_activity"/></list>',
         mockRPC: function (route) {
-            assert.step(route);
+            if (new RegExp('dataset/search_read|init_messaging').test(route)) {
+                assert.step(route);
+            }
```

Comment:
You need to fix this in other tests of this file. Enterprise branch is red because new fetches affect these tests.